### PR TITLE
Update class diagram for Visualization

### PR DESCRIPTION
**Pull request by wwweert123**

### CODE REVIEW

File: docs/diagrams/vis/visualisationClass.puml [hunk 1]
```diff
@@ -1,36 +1,43 @@
...
+
+class "{abstract}\nCommand"  #lime {
++execute() {abstract}
 }
 
-class VisCommand {
-String type
-String chart
-execute()
+class VisCommand #Cyan {
```
Comment (ryan1604) [inline docs/diagrams/vis/visualisationClass.puml]: The colour coding looks nice!

File: docs/diagrams/vis/visualisationClass.puml [hunk 2]
```diff
@@ -1,36 +1,43 @@
...
++<u>sortExpenses(cashflowList: CashflowList)<u>
++<u>sortIncome(cashflowList: CashflowList)<u>
 }
 
-class Visualizer {
-displayChart()
-displayPieChart()
-displayBarChart()
+class Visualizer #Beige {
++<u>displayChart(chartType: String, cashFlowByCat: Map, type: String)<u>
```
Comment (ryan1604) [inline docs/diagrams/vis/visualisationClass.puml]:
```suggestion
 +<u>displayChart(chartType: String, cashFlowByCat: Map, type: String)<u>
```
missing space?

**Review by ryan1604 — APPROVED** [review]
LGTM

File: docs/images/vis/visualisationClass.png
Comment (NeoMinWei) [inline docs/images/vis/visualisationClass.png:1]: what is <"u"> at the end of methods?
the previous format is more legible imo

**Review by NeoMinWei — APPROVED** [review]
LGTM

File: docs/images/vis/visualisationClass.png
Comment (wwweert123) [reply docs/images/vis/visualisationClass.png:1]: Do you mean that I should not include the methods etc.?

**Comment by ryan1604** [comment]
Unrelated but can help to hyperlink the links in the DG?
image

File: docs/images/vis/visualisationClass.png
Comment (NeoMinWei) [reply docs/images/vis/visualisationClass.png:1]: the prev colour scheme easier to read

Comment (wwweert123) [reply docs/images/vis/visualisationClass.png:1]: No wayyyy

**Pull request merged**